src/SearchInput/SearchInput.tsx

```diff
@@ -30,7 +30,7 @@ export interface SearchInputProps extends CommonFieldProps {
   /**  callback to handle found item click */
   onFound: (element: string) => void
```

Comment:
I know `onFound` was probably created before this work started but it seems like this callback is called when an option is selected?

I initially confused `onFound` with results being found, not selected. I think this prop _should_ be changed to something like `onSelectedOption` or `onSelectChange`. Although, probably not something for this PR (as assuming will be breaking change)

Reply:
agreed, I think when we make the breaking change to default fuzzy search, we can make that breaking change as well 🫡